build-support/bin/ci.py

```diff
@@ -450,13 +441,21 @@ def run_jvm_tests() -> None:
 
 def run_integration_tests(*, shard: Optional[str]) -> None:
   check_pants_pex_exists()
+
+  whitelisted_chroot_targets = get_listed_targets("integration_tests_chroot_whitelist.txt")
   all_targets = get_all_python_tests(tag="+integration")
+  non_chrooted_targets = all_targets - whitelisted_chroot_targets
+
   command = ["./pants.pex", "test.pytest"]
   if shard is not None:
     command.append(f"--test-pytest-test-shard={shard}")
   with travis_section("IntegrationTests", f"Running Pants Integration tests {shard if shard is not None else ''}"):
     try:
-      subprocess.run(command + sorted(all_targets) + PYTEST_PASSTHRU_ARGS, check=True)
+      subprocess.run(
```

Comment:
Maybe the output would be easier to parse if chroot/non-chroot were two separate travis sections?